src/api/screenings.ts

```diff
@@ -0,0 +1,10 @@
+import { defaultClient } from '@/api/client';
+
+import type { Screening } from '@/types/screening';
+
+export const getScreeningsByDate = async (
+  date: string
+): Promise<Screening[]> => {
+  const { data } = await defaultClient.get(`/seances?date=${date}`);
+  return data;
+};
```

Comment:
You can provide a suggestion on what to expect directly to axios 😄 
```suggestion
export const getScreeningsByDate = async (
  date: string
) => {
  const { data } = await defaultClient.get<Screening[]>(`/seances?date=${date}`);
  return data;
};
```

Reply:
Ok, thanks for the suggestion, but why is that a better way than typing the function return?
Do I think correctly that it has something to do with passing the type to a higher instance i. e. `defaultClient`?

Reply:
It makes the `data` be typed the way we want and then the rest is left for TypeScript to infer the type 😄 as it knows everything - otherwise the `data` is by default of `any` type

Reply:
When You put it this way it makes sense, well noted :bowtie: